src/main.rs

```diff
@@ -84,6 +84,7 @@ fn setup(config_folder: &Path) -> IoResult<(Data<RwLock<Config>>, ServerConfig,
                         if let Err(e) = config_sender.send(config) {
                             log::error!("Failed to send config for the cleanup routine: {}", e)
                         }
+                        cloned_config.warn_deprecation();
```

Comment:
Good catch here!

Reply:
Yea, I forgot this previously, but noticed it when testing the code manually.